testsuite/forge/src/runner.rs

```diff
@@ -141,7 +141,7 @@ impl<'cfg, F: Factory> Forge<'cfg, F> {
         summary.write_starting_msg()?;
 
         let mut rng = ::rand::rngs::StdRng::from_seed(OsRng.gen());
-        let mut swarm = self.factory.launch_swarm(1);
+        let mut swarm = self.factory.launch_swarm(9);
```

Comment:
Lets drop this one line for now and we can later look at having this be either a config or something.

Reply:
agree, we should parametric some hardcode parameters